tutorials/run-laravel-on-appengine-flexible/index.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Run Laravel on Google App Engine Flexible Environment
+description: Learn how to deploy a Laravel app to Google App Engine flexible environment.
+author: bshaffer
+tags: App Engine, Laravel, PHP
+date_published: 2017-03-15
+---
+## Laravel
+
+> [Laravel][laravel]: The PHP Framework For Web Artisans.
+>
+> – laravel.com
+
+You can check out [PHP on Google Cloud Platform][php-gcp] to get an
+overview of PHP itself and learn ways to run PHP apps on Google Cloud
+Platform.
+
+## Prerequisites
+
+1. Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com/).
+1. Enable billing for your project.
+1. Install the [Google Cloud SDK](https://cloud.google.com/sdk/).
+
+## Prepare
+
+1. Follow the official documentation for [installing Laravel][laravel-install]
+from laravel.com.
+
+## Run
+
+1. Run the app with the following command:
+
+        php artisan serve
+
+1. Visit [http://localhost:8000](http://localhost:8000) to see the Laravel
+Welcome page.
+
+## Deploy
+
+1. Create an `app.yaml` file with the following contents:
+
+        runtime: php
+        env: flex
+
+        runtime_config:
+          document_root: public
+
+        # required on some platforms so ".env" is not skipped
+        skip_files: false
+
+        env_variables:
+          # The values here will override those in ".env". This is useful for
+          # production-specific configuration. However, feel free to set these
+          # values in ".env" instead if you prefer.
+          APP_LOG: errorlog
+          STORAGE_DIR: /tmp
+
+1. Add the following under "scripts" in `composer.json`:
+
+        "post-deploy-cmd": [
+            "chmod -R 755 bootstrap\/cache"
+        ]
+
+    In the context of Laravel's `composer.json`, it will look like this:
+    ![Add post-deploy-cmd scripts to composer.json][composer-json]
+
+1. Run the following command to deploy your app:
+
+        gcloud app deploy
+
+1. Visit `http://YOUR_PROJECT_ID.appspot.com` to see the Laravel welcome page!
+
+    ![Laravel welcome page][laravel-welcome]
+
+## Set up Database Sessions
+
+1. Follow the instructions to set up a
```

Comment:
Tell users whether to use MySQL or PostgreSQL. (I recommend MySQL, since PostgreSQL is in beta.)

Also, missing period at the end of the sentence.